server/server.go

```diff
@@ -711,17 +734,18 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 		commitStatusUpdater,
 		projectCommandBuilder,
 		prjCmdRunner,
-		pullUpdater,
+		outputUpdater,
 		dbUpdater,
 		userConfig.ParallelPoolSize,
 		pullReqStatusFetcher,
 	)
 
+	// Using pull updater for approving policies until we move off of PR comments entirely
 	approvePoliciesCommandRunner := events.NewApprovePoliciesCommandRunner(
 		commitStatusUpdater,
 		projectCommandBuilder,
 		prjCmdRunner,
-		pullUpdater,
+		&pullOutputUpdater,
```

Comment:
Why? We should move these to github checks as well.

Reply:
Hmm I thought it would make more sense to comment out that `policies were approved `and so on, but since we'll be moving off of PR comments entirely, it makes sense to use github checks for all the command runners. I've replaced it with outputUpdater.